rotkehlchen/chain/evm/decoding/aura_finance/decoder.py

```diff
@@ -0,0 +1,240 @@
+
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+            claim_zap_address: ChecksumEvmAddress,
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+        )
+        self.claim_zap_address = claim_zap_address
+
+    def _decode_lock_aura(self, context: DecoderContext) -> DecodingOutput:
+        """Decodes locking AURA events on Ethereum and Base (vlAURA)."""
+        locked_amount = token_normalized_value_decimals(
+            token_amount=int.from_bytes(context.tx_log.data[32:64]),
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )
+        for event in context.decoded_events:
+            if (
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.balance.amount == locked_amount
+            ):
+                event.counterparty = CPT_AURA_FINANCE
+                event.event_type = HistoryEventType.DEPOSIT
+                event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                event.notes = f'Lock {locked_amount} {event.asset.resolve_to_asset_with_symbol().symbol} in auraLocker (vlAURA)'  # noqa: E501
+
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_lock_aura_bridged(self, context: DecoderContext) -> DecodingOutput:
+        """Decodes locking AURA events on bridged chains (excluding Ethereum)."""
+        locked_amount = token_normalized_value_decimals(
+            token_amount=int.from_bytes(context.tx_log.data[:32]),
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )
+        refund_event = None
+        for event in context.decoded_events:
+            if (
+                event.event_type == HistoryEventType.RECEIVE and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.asset == self.evm_inquirer.native_token
+            ):  # Refund event (if any) comes first
+                refund_event = event
+            if (
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.asset == self.evm_inquirer.native_token
+            ):
+                event.counterparty = CPT_AURA_FINANCE
+                event.event_subtype = HistoryEventSubType.FEE
+                # Calculate bridge fee, accounting for potential refunds
+                actual_bridge_fee = event.balance.amount if refund_event is None else event.balance.amount - refund_event.balance.amount  # noqa: E501
+                event.balance.amount = actual_bridge_fee
+                event.notes = f'Pay {actual_bridge_fee} {self.evm_inquirer.native_token.symbol} as bridge fee (to Ethereum)'  # noqa: E501
```

Comment:
why hard coded ethereum here?

Reply:
that's the only possible destination at the moment.